modules-disabled/architecture/pages/pubcloud-setup.adoc

```diff
@@ -1,20 +1,23 @@
 [[public-cloud-setup]]
 = Public Cloud Setup
 
-{productname} is pre-installed on images that are available in the Public Cloud. 
+// Deprecated. See Quick Start: Public Cloud Guide instead. --LKB 2021-01-13
```

Comment:
Same: do not add comments, they may break translations (not always but there is no reason do add this comment).